api/swagger.yaml

```diff
@@ -720,7 +720,15 @@ definitions:
             description: "Gives the container full access to the host."
           PublishAllPorts:
             type: "boolean"
-            description: "Allocates a random host port for all of a container's exposed ports."
+            description: |
+              Allocates an ephemeral host port for all of a container's
+              exposed ports. Specified as a boolean value.
```

Comment:
I don't think you need the note about it being boolean here, since it's defined in line 722 and that will make Swagger print it out in the resulting doc.